server/deps.edn

```diff
@@ -95,14 +95,17 @@
                             ;; print stack traces instead of "Full report at ..."
                             "-Dclojure.main.report=stderr"
                             "-Dclojure.server.repl={:port 6006 :accept clojure.core.server/repl}"
-                            
+
                             ;; hazelcast
                             "--add-modules" "java.se"
                             "--add-exports" "java.base/jdk.internal.ref=ALL-UNNAMED"
                             "--add-opens" "java.base/java.lang=ALL-UNNAMED"
                             "--add-opens" "java.base/sun.nio.ch=ALL-UNNAMED"
                             "--add-opens" "java.management/sun.management=ALL-UNNAMED"
-                            "--add-opens" "jdk.management/com.sun.management.internal=ALL-UNNAMED"]}
+                            "--add-opens" "jdk.management/com.sun.management.internal=ALL-UNNAMED"
+
+                            ;; headless charts 
+                            "-Djava.awt.headless=true"]}
```

Comment:
Can you add it to the [docker-compose.yml](https://github.com/instantdb/instant/blob/main/server/docker-compose.yml#L18) also

Reply:
Maybe should also add it to the test flags in the deps.edn?

Reply:
🫡 

I ran `make test`, and curiously the little guy did not show up. I will add it in for good measure though 🫡